scipy/interpolate/_cubic.py

```diff
@@ -538,13 +538,24 @@ def __init__(self, x, y, axis=0, *, method: Literal["akima", "makima"]="akima",
             else:
                 f1 = dm[2:]
                 f2 = dm[:-2]
+
+            # makima is more numerically stable for small f12,
+            # so a finite cutoff should not improve any behavior
+            # however, akima has a qualitative discontinuity near f12=0
```

Comment:
Not sure what a qualitative discontinuity is. Rephrasing is probably not worth the churn by itself, but if there's an unrelated commit/push, it'd be nice to clarify.